### PR TITLE
adding schema setup + teardown files

### PR DESCRIPTION
**Pull request by mabbason**
I included the ideas in @scogra17 most recent PR regarding inserting seed-data. I also tore-down and then setup DB so that it reflects the schema in this PR.

### CODE REVIEW

**Review by katarinarosiak — APPROVED** [review]

**Comment by katarinarosiak** [comment]
Great Job Miles!

**Review by tdronkers04 — APPROVED** [review]

**Review by tdronkers04 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
 # About
-App for performing synthetic test CRUD operations
\ No newline at end of file
+App for performing synthetic test CRUD operations
+
+# Sample Git Text for Co-Authoring a Commit in CLI
```
Comment (scogra17) [inline README.md:4]: Have you tried git mob?

File: sql/schema.sql [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  run_frequency_mins INT NOT NULL,
+  method text NOT NULL,
+  url text NOT NULL,
+  headers JSONB,
+  payload JSONB,
+  query_params JSONB,
+  teardown text,
+  status text NOT NULL,
+  created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+  updated_at TIMESTAMP NOT NULL
```
Comment (scogra17) [inline sql/schema.sql]: What do you think about removing the NOT NULL constraint on `updated_at`? That way we can leave null when a test is first created and populate on the first update

**Review by scogra17 — APPROVED** [review]
Added a couple comments

File: README.md (hunk 1 above)
Comment (mabbason) [reply README.md:4]: I have not yet, it's a little ways down my to-do list but I could try to squeeze it in ;)

**Pull request merged**